frontend/src/lib/components/CommandPalette/commandLogic.ts

```diff
@@ -229,14 +235,21 @@ export const commandLogic = kea<commandLogicType<Command, CommandRegistrations>>
                     }
                     resolveCommand(command, directResults, argument)
                 }
-                const fuse = new Fuse(directResults.concat(prefixedResults), {
+                const allResults = directResults.concat(prefixedResults)
+                const fusableResults: CommandResult[] = []
+                const guaranteedResults: CommandResult[] = []
+                for (const result of allResults) {
+                    if (result.showAlways) guaranteedResults.push(result)
+                    else fusableResults.push(result)
+                }
+                const fusedResults = new Fuse(fusableResults, {
                     keys: ['display', 'synonyms'],
                 })
-                return fuse
                     .search(argument)
                     .slice(0, RESULTS_MAX)
                     .map((result) => result.item)
-                    .sort((result) => (result.command.scope === GLOBAL_COMMAND_SCOPE ? 1 : -1))
+                const finalResults = guaranteedResults.concat(fusedResults)
+                return finalResults.sort((result) => (result.command.scope === GLOBAL_COMMAND_SCOPE ? 1 : -1))
```

Comment:
Nit: this seems to work, though the `sort` function in JS takes two arguments. Right now with just comparing the first argument's scope with `"global"` is probably slightly less efficient than if you'd return 0 if the first and second argument's scopes are equal.